lib/export-csv.js

```diff
@@ -15,21 +14,42 @@ function cmd_export_dojo_users (args, callback) {
     deleted: 0,
     limit$: 'NULL'
   };
-  var csvFields = ['Name', 'Email', 'Type'];
+  var csvFields = ['Name', 'Email', 'Type', 'Parent Emails'];
+
   function convertToCSV (csvData, done) {
     json2csv({data: csvData, fields: csvFields}, done);
   }
+  // Returns the parent email of a child
+  function getParentEmail (csvData, done) {
+    // filter to get children
+    async.eachSeries(csvData, function (row, cb) {
+      if (_.includes(row.Type, 'attendee')) {
+        seneca.act({role: 'cd-profiles', cmd: 'load_parents_for_user', userId: row.UserId, user: args.user}, function (err, parents) {
+          if (err) return callback(err);
```

Comment:
@DanielBrierton if error, returning the act cb inside a each may create a timeout?